### ISSUE TITLE
Access faces of a mesh based on their ouput colour

### ISSUE THREAD

**Issue by priyabiswas12** [marcomusy/vedo#1156]
![image](https://github.com/user-attachments/assets/092fa7a9-d762-4161-aaec-51e8710d6973)

Hi there, 

I have a mesh whose faces id like to access based on the output colour as seen above. For example, id like to access the traingles in blue in the above example. Is there a bulit-in method to do this? Thank you

**Comment by marcomusy** [comment]
Yes with [add_ids](https://vedo.embl.es/autodocs/content/vedo/vedo/core.html#CommonAlgorithms.add_ids)
```py
mesh.add_ids()
print(mesh.pointdata["PointID"])
print(mesh.pointdata["Gauss_Curvature"])
```

**Comment by priyabiswas12** [comment]
sorry, i meant how do I retrieve the ID of the triangles shaded in blue in the above example?

**Comment by marcomusy** [comment]
You can associate data to points (vertices) or to faces (cells) in vedo.
Curvature is computed on points. You can interpolate it to triangle faces with `map_points_to_cells()` [here](https://vedo.embl.es/autodocs/content/vedo/vedo/core.html#CommonAlgorithms.map_points_to_cells).

Once you have them you can threshold your arrays with normal numpy commands.
Let me know if it is not clear.
